### DISCUSSION TITLE
Doris 2.3物化视图无法创建，加上where条件后又能创建了。

### DISCUSSION THREAD

**Discussion by DA1OOO** [apache/doris#29025] [A - General / Q&A] [answered]
我的表中的数据是7-12月，按天分区。
`create material view as select xxx from table group by xxx` 会报如下错误：
errCode = 2, detailMessage = rollup tasks failed on same tablet reach threshold [[ALTER], signature: 627801, backendId: 12320, tablet id: 627801, [ALTER], signature: 627801, backendId: 12290, tablet id: 627801], reason=task type: ALTER, status_code: INTERNAL_ERROR, status_message: [()[E-1602]SchemaChange check row nums failed], backendId: Backend [id=12290, host=, heartbeatPort=9050, alive=true, lastStartTime=2023-12-24 22:01:31, tags: {location=default}], signature: 627801

然后我在建物化视图的语句加上过滤条件之后，又能正常创建：
`create material view as select xxx from table where d_s > '20230630' group by xxx`
这两个创建语句读取的数据量应该都是相同的，请问为什么会出现这个原因。

**Comment by LemonLiTree** [comment, marked as answer]
https://github.com/apache/doris/pull/28938
fixed
